### PR TITLE
test improvements to make test results looking alike in percy and eye…

### PR DESCRIPTION
**Pull request by dmytrofirsov**
test improvements to make test results looking alike in percy and eyes + fix for the NUI-5945

### CODE REVIEW

File: packages/bits/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "build-examples": "ng b demo",
     "prebuild-examples": "npm run compile-colors",
     "build-examples:prod": "npm run build-examples --prod",
-    "build-examples:watch": "npm run build-examples --watch",
+    "build-examples:watch": "ng b demo --watch",
```
Comment (Vita-Karas) [inline packages/bits/package.json:41]: Do we need the same changes for `build-examples:prod` script?

Comment (dmytrofirsov) [reply packages/bits/package.json:41]: looks like not, it fails builds.

**Review by NataliiaPodpriatova — DISMISSED** [review]

File: packages/bits/spec/virtual-camera/types.ts [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+    Eyes = "eyes",
+    Percy = "percy"
 }
 
 export const LENSES: string[] =  [ ...Object.values(LensType) ];
 
 export const CAMERA_DEFAULT_SETTINGS: ICameraSettings = {
     fullframe: true,
-    responsive: [1920],
+    responsiveWiths: [1920],
```
Comment (Blakesalot) [inline packages/bits/spec/virtual-camera/types.ts:12]:
```suggestion
    responsiveWidths: [1920],
```

File: packages/bits/spec/virtual-camera/CameraEngine.ts [hunk 3]
```diff
@@ -23,10 +23,10 @@ export class CameraEngine {
         if (this.settings.globalLens && LENSES.includes(this.settings.globalLens)) {
```
Comment (Blakesalot) [inline packages/bits/spec/virtual-camera/CameraEngine.ts:23]: I just noticed that the filename doesn't follow the kebab-case naming convention (`camera-engine.ts`)

File: packages/bits/spec/virtual-camera/CameraSettings.ts [hunk 4]
```diff
@@ -19,10 +19,10 @@ export class CameraSettings {
     private async crop(): Promise<void> { this.currentSettings.fullframe = false }
```
Comment (Blakesalot) [inline packages/bits/spec/virtual-camera/CameraSettings.ts:19]: I just noticed that the filename doesn't follow the kebab-case naming convention

File: packages/bits/spec/virtual-camera/EyesLens.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@ import { getCurrentBranchName } from "../helpers";
 export class EyesLens implements ILens {
```
Comment (Blakesalot) [inline packages/bits/spec/virtual-camera/EyesLens.ts:4]: I just noticed that the filename doesn't follow the kebab-case naming convention

File: packages/bits/spec/virtual-camera/PercyLens.ts [hunk 6]
```diff
@@ -27,7 +27,7 @@ export class PercyLens implements ILens {
     public toolConfig() { console.warn("No config is availalbe for Percy"); }
```
Comment (Blakesalot) [inline packages/bits/spec/virtual-camera/PercyLens.ts:27]: I just noticed that the filename doesn't follow the kebab-case naming convention

File: packages/bits/spec/virtual-camera/types.ts (hunk 2 above)
Comment (dmytrofirsov) [reply packages/bits/spec/virtual-camera/types.ts]: fixed

File: packages/bits/spec/virtual-camera/CameraEngine.ts (hunk 3 above)
Comment (dmytrofirsov) [reply packages/bits/spec/virtual-camera/CameraEngine.ts:23]: fixed for all affected files

**Review by Blakesalot — APPROVED** [review]

**Review by NataliiaPodpriatova — APPROVED** [review]

**Pull request merged**